lerobot/common/datasets/lerobot_dataset.py

```diff
@@ -36,7 +36,7 @@
 from lerobot.common.datasets.video_utils import VideoFrame, load_from_videos
 
 DATA_DIR = Path(os.environ["DATA_DIR"]) if "DATA_DIR" in os.environ else None
-CODEBASE_VERSION = "v1.4"
+CODEBASE_VERSION = "v1.5"
```

Comment:
@Cadene is this what I should be doing?

Reply:
@alexander-soare You should revert this and instead probably create a new pusht dataset that doesnt contain images but environment_state.

What do you think?

Reply:
See 6b46e6a3a8a7a02e335abd897766c46b2bf742b6

Reply:
Agreed. We should update the version ;)